### ISSUE TITLE
[BUG] TRANSPARENT block IDs

### ISSUE THREAD

**Issue by 100petr** [LoneDev6/ItemsAdder#437]
**Describe the bug**
REAL_TRANSPARENT block IDs has a negative number.

**To Reproduce**
1. Start server
2. See console

**Expected behavior**
REAL_TRANSPARENT block IDs will be a positive number.

**Screenshots**
![image](https://user-images.githubusercontent.com/29830635/97112730-2ee54a80-16e6-11eb-8aa2-5630e90993ec.png)

**Info (please complete the following information):**
 - Server version: paper 1.16.3
 - Plugin version: 2.1.10

**Additional context**
In your opinion, this is just a visual error.

**Comment by LoneDev6** [comment]
Thanks, will be fixed

**Comment by mA0w0** [comment]
2.1.13
paper251

When I updated to the latest version, I also started to see it.

The already generated file is fine, but when I add a new block and the custom model data json is generated, the numbers are negative and are not applied correctly. Even if you specify the model ID and generate it, it was a negative number, so I think that automatic generation will probably be the same.

![f4302b0ec2061505ee35df0f45811b63](https://user-images.githubusercontent.com/72402669/97733848-b33b3180-1b1b-11eb-94aa-8be7d55d7b90.png)
